src/store/epics/sign.ts

```diff
@@ -1,21 +1,23 @@
-import { verifyMessage } from '@actions'
+import { messageVerificationResult } from '@actions'
 import { generalFailureAlert, generalSuccessAlert } from '@helpers/alert'
 import { Epic } from '@store'
 import { fromPromise } from 'rxjs/observable/fromPromise'
 import { filter, ignoreElements, map } from 'rxjs/operators'
 import { isActionOf } from 'typesafe-actions'
 
-export const verificationNotification$: Epic = (action$) =>
+export const verificationNotification$: Epic = action$ =>
   action$.pipe(
-    filter(isActionOf(verifyMessage)),
-    map(({payload}) => {
+    filter(isActionOf(messageVerificationResult)),
+    map(({ payload }) => {
       if (payload) {
         return fromPromise(generalSuccessAlert('The signature is valid.'))
```

Comment:
this tells me that the signature is the important part, rather than the message, we could also make it clearer what we're receiving by renaming the payload back to its inputted value of `isValid`?

Reply:
Definitely would be nicer.